lib/project_types/script/messages/messages.rb

```diff
@@ -148,6 +145,12 @@ module Messages
           language_library_for_api_not_found_cause: "Script can’t be pushed because the %{language} library for API %{api} is missing.",
           language_library_for_api_not_found_help: "Make sure extension_point.yml contains the correct API library.",
           no_scripts_found_in_app: "The selected apps have no scripts. Please, create them first on the partners' dashboard.",
+
+          missing_env_file_variables: "The following variables are missing in the .env file: %s."\
```

Comment:
@MeredithCastile we need your help here :). We print the former when there are missing variables in the `.env` file, and the latter when the user calls the push command from a CI environment and forgets one of the required options, for example `--api-key`:

Here's an example:

![image](https://user-images.githubusercontent.com/663605/148398916-93d0ac6c-ace4-4213-94e4-9b4f3707573b.png)